README.md

```diff
@@ -24,6 +24,19 @@ regions which works alongside the main renderer, and mouse support.
 To get started, see the tutorial below, the [examples][examples], the
 [docs][docs] and some common [resources](#libraries-we-use-with-bubble-tea).
 
+Note: If you're working with submodels, you don't need to worry about using commands to send data to your main model as the data flows downward. 
```

Comment:
I wonder if it's too early to introduce both this and the section below about lifecycle, the reasoning being that new users will see details here before having an understanding of how the framework works. Perhaps there's a general `README.md` in the `docs` directory where we can cover things like this? It would probably be good to have a README in there regardless to greet people as they step in.

Reply:
I agree

Reply:
I moved this to docs/README which will continue to grow as I gather more info on the frequently asked questions in issues, etc